runtime/vimrc_example.vim

```diff
@@ -1,24 +1,14 @@
 " An example for a vimrc file.
 "
-" Maintainer:   Bram Moolenaar <Bram@vim.org>
-" Last change:  2014 Nov 05
-"
 " To use it, copy it to
 "     for Unix:  ~/.vimrc
-"     for MS-DOS and Win32:  $VIM\_vimrc
-
-" allow backspacing over everything in insert mode
-set backspace=indent,eol,start
+"     for Windows:  $VIM\_vimrc
 
 set backup             " keep a backup file (restore to previous version)
 set undofile           " keep an undo file (undo changes after closing)
 set history=50         " keep 50 lines of command line history
```

Comment:
This is already the default, so maybe preemptively remove it? I say so because you have a PR which changes the default IIRC.